src/components/shared/ManagedTree.js

```diff
@@ -44,10 +44,6 @@ class ManagedTree extends Component {
       expanded: new Set(),
       focusedItem: null
     };
-
-    const self: any = this;
```

Comment:
I don't think this is dead code. These are binding the methods to the prototype.